src/pages/ReportSettingsPage.js

```diff
@@ -55,6 +55,8 @@ class ReportSettingsPage extends Component {
 
         this.validate = this.validate.bind(this);
         this.updatePolicyRoomName = this.updatePolicyRoomName.bind(this);
+        this.shouldDisableRename = ReportUtils.isDefaultRoom(this.props.report)
+        || ReportUtils.isArchivedRoom(this.props.report);
```

Comment:
NAB: You can probably put this on one line
```suggestion
        this.shouldDisableRename = ReportUtils.isDefaultRoom(this.props.report) || ReportUtils.isArchivedRoom(this.props.report);
```